[2020-06-10 16:22:47] INFO [<module>: config.py, 11] hello
[2020-06-10 16:28:59] INFO [<module>: config.py, 11] hello
[2020-06-10 16:28:59] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:80
[2020-06-10 16:28:59] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:80 "POST /cms/api/user/login HTTP/1.1" 200 None
[2020-06-10 16:28:59] INFO [test_user_login_noraml: test_user_login.py, 38] 测试用例：test_user_login_normal
[2020-06-10 16:28:59] INFO [test_user_login_noraml: test_user_login.py, 39] url：http://172.16.170.49/cms/api/user/login
[2020-06-10 16:28:59] INFO [test_user_login_noraml: test_user_login.py, 40] 请求参数：{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}
[2020-06-10 16:28:59] INFO [test_user_login_noraml: test_user_login.py, 41] 期望结果：{"status":true,"code":200,"message":null,"data":{"sn":"test","name":"柴珍珍","indexUrl":null,"headUrl":null,"roles":[{"id":1,"name":"超级管理员","code":"000001"}],"initPassword":0},"otherData":{}}
[2020-06-10 16:28:59] INFO [test_user_login_noraml: test_user_login.py, 42] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\')\n at [Source: java.io.PushbackInputStream@2e5ff63e; line: 1, column: 1]'}}
[2020-06-10 16:28:59] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:80
[2020-06-10 16:28:59] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:80 "POST /cms/api/user/login HTTP/1.1" 200 None
[2020-06-10 16:28:59] INFO [test_user_login_wrong: test_user_login.py, 56] 测试用例：test_user_login_normal
[2020-06-10 16:28:59] INFO [test_user_login_wrong: test_user_login.py, 57] url：http://172.16.170.49/cms/api/user/login
[2020-06-10 16:28:59] INFO [test_user_login_wrong: test_user_login.py, 58] 请求参数：{"sn":"test","password":"1234567"}
[2020-06-10 16:28:59] INFO [test_user_login_wrong: test_user_login.py, 59] 期望结果：{"status":false,"code":0,"message":"用户名或密码错误，请重新输入","data":null,"otherData":{}}
[2020-06-10 16:28:59] INFO [test_user_login_wrong: test_user_login.py, 60] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\')\n at [Source: java.io.PushbackInputStream@746edf1c; line: 1, column: 1]'}}
[2020-06-11 11:25:08] DEBUG [<module>: run_all.py, 8] =======================测试开始=======================
[2020-06-11 11:25:08] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 11:25:08] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 11:25:08] INFO [test_user_login_noraml: test_user_login.py, 38] 测试用例：test_user_login_normal
[2020-06-11 11:25:08] INFO [test_user_login_noraml: test_user_login.py, 39] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 11:25:08] INFO [test_user_login_noraml: test_user_login.py, 40] 请求参数：{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}
[2020-06-11 11:25:08] INFO [test_user_login_noraml: test_user_login.py, 41] 期望结果：{"status":true,"code":200,"message":null,"data":{"sn":"test","name":"柴珍珍","indexUrl":null,"headUrl":null,"roles":[{"id":1,"name":"超级管理员","code":"000001"}],"initPassword":0},"otherData":{}}
[2020-06-11 11:25:08] INFO [test_user_login_noraml: test_user_login.py, 42] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\')\n at [Source: java.io.PushbackInputStream@1cf0c9c1; line: 1, column: 1]'}}
[2020-06-11 11:25:08] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 11:25:08] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 11:25:08] INFO [test_user_login_wrong: test_user_login.py, 56] 测试用例：test_user_login_normal
[2020-06-11 11:25:08] INFO [test_user_login_wrong: test_user_login.py, 57] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 11:25:08] INFO [test_user_login_wrong: test_user_login.py, 58] 请求参数：{"sn":"test","password":"1234567"}
[2020-06-11 11:25:08] INFO [test_user_login_wrong: test_user_login.py, 59] 期望结果：{"status":false,"code":0,"message":"用户名或密码错误，请重新输入","data":null,"otherData":{}}
[2020-06-11 11:25:08] INFO [test_user_login_wrong: test_user_login.py, 60] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\')\n at [Source: java.io.PushbackInputStream@267dafb0; line: 1, column: 1]'}}
[2020-06-11 11:25:30] ERROR [send_email: send_email.py, 25] [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
[2020-06-11 15:14:29] DEBUG [<module>: run_all.py, 8] =======================测试开始=======================
[2020-06-11 15:14:29] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:14:29] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:14:29] INFO [test_user_login_noraml: test_user_login.py, 41] 测试用例：test_user_login_normal
[2020-06-11 15:14:29] INFO [test_user_login_noraml: test_user_login.py, 42] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:14:29] INFO [test_user_login_noraml: test_user_login.py, 43] 请求参数：{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}
[2020-06-11 15:14:29] INFO [test_user_login_noraml: test_user_login.py, 44] 期望结果：{"status":true,"code":200,"message":null,"data":{"sn":"test","name":"柴珍珍","indexUrl":null,"headUrl":null,"roles":[{"id":1,"name":"超级管理员","code":"000001"}],"initPassword":0},"otherData":{}}
[2020-06-11 15:14:29] INFO [test_user_login_noraml: test_user_login.py, 45] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\')\n at [Source: java.io.PushbackInputStream@6609bd2d; line: 1, column: 1]'}}
[2020-06-11 15:14:30] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:14:30] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:14:30] INFO [test_user_login_wrong: test_user_login.py, 59] 测试用例：test_user_login_normal
[2020-06-11 15:14:30] INFO [test_user_login_wrong: test_user_login.py, 60] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:14:30] INFO [test_user_login_wrong: test_user_login.py, 61] 请求参数：{"sn":"test","password":"1234567"}
[2020-06-11 15:14:30] INFO [test_user_login_wrong: test_user_login.py, 62] 期望结果：{"status":false,"code":0,"message":"用户名或密码错误，请重新输入","data":null,"otherData":{}}
[2020-06-11 15:14:30] INFO [test_user_login_wrong: test_user_login.py, 63] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\')\n at [Source: java.io.PushbackInputStream@35c24381; line: 1, column: 1]'}}
[2020-06-11 15:14:51] ERROR [send_email: send_email.py, 26] [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
[2020-06-11 15:17:06] DEBUG [<module>: run_all.py, 8] =======================测试开始=======================
[2020-06-11 15:17:06] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:17:06] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:17:06] INFO [test_user_login_noraml: test_user_login.py, 41] 测试用例：test_user_login_normal
[2020-06-11 15:17:06] INFO [test_user_login_noraml: test_user_login.py, 42] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:17:06] INFO [test_user_login_noraml: test_user_login.py, 43] 请求参数：{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}
[2020-06-11 15:17:06] INFO [test_user_login_noraml: test_user_login.py, 44] 期望结果：{"status":true,"code":200,"message":null,"data":{"sn":"test","name":"柴珍珍","indexUrl":null,"headUrl":null,"roles":[{"id":1,"name":"超级管理员","code":"000001"}],"initPassword":0},"otherData":{}}
[2020-06-11 15:17:06] INFO [test_user_login_noraml: test_user_login.py, 45] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\')\n at [Source: java.io.PushbackInputStream@104a08c1; line: 1, column: 1]'}}
[2020-06-11 15:17:06] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:17:06] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:17:06] INFO [test_user_login_wrong: test_user_login.py, 59] 测试用例：test_user_login_password_wrong
[2020-06-11 15:17:06] INFO [test_user_login_wrong: test_user_login.py, 60] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:17:06] INFO [test_user_login_wrong: test_user_login.py, 61] 请求参数：{"sn":"test","password":"1234567"}
[2020-06-11 15:17:06] INFO [test_user_login_wrong: test_user_login.py, 62] 期望结果：{"status":false,"code":0,"message":"用户名或密码错误，请重新输入","data":null,"otherData":{}}
[2020-06-11 15:17:06] INFO [test_user_login_wrong: test_user_login.py, 63] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\')\n at [Source: java.io.PushbackInputStream@31f0b6; line: 1, column: 1]'}}
[2020-06-11 15:17:27] ERROR [send_email: send_email.py, 26] [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
[2020-06-11 15:32:13] DEBUG [<module>: run_all.py, 8] =======================测试开始=======================
[2020-06-11 15:32:13] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:32:13] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:32:13] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:32:13] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:32:13] INFO [test_user_login_wrong: test_user_login.py, 62] 测试用例：test_user_login_password_wrong
[2020-06-11 15:32:13] INFO [test_user_login_wrong: test_user_login.py, 63] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:32:13] INFO [test_user_login_wrong: test_user_login.py, 64] 请求参数：{"sn":"test","password":"1234567"}
[2020-06-11 15:32:13] INFO [test_user_login_wrong: test_user_login.py, 65] 期望结果：{"status":false,"code":0,"message":"用户名或密码错误，请重新输入","data":null,"otherData":{}}
[2020-06-11 15:32:13] INFO [test_user_login_wrong: test_user_login.py, 66] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\')\n at [Source: java.io.PushbackInputStream@79fc1b89; line: 1, column: 1]'}}
[2020-06-11 15:32:34] ERROR [send_email: send_email.py, 26] [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
[2020-06-11 15:33:32] DEBUG [<module>: run_all.py, 8] =======================测试开始=======================
[2020-06-11 15:33:32] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:33:32] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:33:32] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:33:32] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:33:32] INFO [test_user_login_wrong: test_user_login.py, 63] 测试用例：test_user_login_password_wrong
[2020-06-11 15:33:32] INFO [test_user_login_wrong: test_user_login.py, 64] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:33:32] INFO [test_user_login_wrong: test_user_login.py, 65] 请求参数：{"sn":"test","password":"1234567"}
[2020-06-11 15:33:32] INFO [test_user_login_wrong: test_user_login.py, 66] 期望结果：{"status":false,"code":0,"message":"用户名或密码错误，请重新输入","data":null,"otherData":{}}
[2020-06-11 15:33:32] INFO [test_user_login_wrong: test_user_login.py, 67] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\')\n at [Source: java.io.PushbackInputStream@6abb6fa6; line: 1, column: 1]'}}
[2020-06-11 15:33:53] ERROR [send_email: send_email.py, 26] [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
[2020-06-11 15:34:06] DEBUG [<module>: run_all.py, 8] =======================测试开始=======================
[2020-06-11 15:34:06] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:34:06] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:34:06] INFO [test_user_login_noraml: test_user_login.py, 40] 测试用例：test_user_login_normal
[2020-06-11 15:34:06] INFO [test_user_login_noraml: test_user_login.py, 41] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:34:06] INFO [test_user_login_noraml: test_user_login.py, 42] 请求参数：{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}
[2020-06-11 15:34:06] INFO [test_user_login_noraml: test_user_login.py, 43] 期望结果：{"status":true,"code":200,"message":null,"data":{"sn":"test","name":"柴珍珍","indexUrl":null,"headUrl":null,"roles":[{"id":1,"name":"超级管理员","code":"000001"}],"initPassword":0},"otherData":{}}
[2020-06-11 15:34:06] INFO [test_user_login_noraml: test_user_login.py, 44] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"D5BCF1D75F6C61A276B0701AC2933765"}\')\n at [Source: java.io.PushbackInputStream@55f4d703; line: 1, column: 1]'}}
[2020-06-11 15:34:06] DEBUG [_new_conn: connectionpool.py, 226] Starting new HTTP connection (1): 172.16.170.49:12345
[2020-06-11 15:34:06] DEBUG [_make_request: connectionpool.py, 433] http://172.16.170.49:12345 "POST http://172.16.170.49/cms/api/user/login HTTP/1.1" 200 None
[2020-06-11 15:34:06] INFO [test_user_login_wrong: test_user_login.py, 58] 测试用例：test_user_login_password_wrong
[2020-06-11 15:34:06] INFO [test_user_login_wrong: test_user_login.py, 59] url：http://172.16.170.49/cms/api/user/login
[2020-06-11 15:34:06] INFO [test_user_login_wrong: test_user_login.py, 60] 请求参数：{"sn":"test","password":"1234567"}
[2020-06-11 15:34:06] INFO [test_user_login_wrong: test_user_login.py, 61] 期望结果：{"status":false,"code":0,"message":"用户名或密码错误，请重新输入","data":null,"otherData":{}}
[2020-06-11 15:34:06] INFO [test_user_login_wrong: test_user_login.py, 62] 实际结果：{'status': False, 'code': 50021, 'message': '参数转换异常', 'data': None, 'otherData': {'detail': 'JSON parse error: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\'); nested exception is com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.xdbigdata.usercenter.model.dto.UserLoginDto: no String-argument constructor/factory method to deserialize from String value (\'{"sn":"test","password":"1234567"}\')\n at [Source: java.io.PushbackInputStream@1ad41739; line: 1, column: 1]'}}
[2020-06-11 15:34:27] ERROR [send_email: send_email.py, 26] [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
